static/i18n/en/app.yml

```diff
@@ -283,24 +283,26 @@ settings:
     about: Help
   display:
     desc:
-    language: Language
-    languageDesc: Choose the language to display.
+    language: Display language
+    languageDesc: Set the language displayed in Ledger Live.
     counterValue: Base currency
-    counterValueDesc: Choose the currency to display next to your balance and operations. An exchange is used (via BTC pair).
+    counterValueDesc: Choose the currency to display next to your balance and operations.
```

Comment:
same here, either put it back but to me we miss a description for the exchange that is used. (see on settings screen)

Reply:
ultimately, maybe we need to have 2 settings like we used to have..